content/technotes/2018-08-08-phylotar-release.md

```diff
@@ -0,0 +1,173 @@
+---
+slug: "phylotar"
+title: "phylotaR: Retrieve Orthologous Sequences from GenBank"
+package_version: 1.0.0
+authors:
+  - name: Dom Bennett
+    url: https://github.com/dombennett
+date: 2018-08-08
+categories: technote
+topicid: 1256
+tags:
+- r
+- ropensci
+- community
+- software
+- review
+- onboarding
+- package
+- evolution
+- bioinformatics
+- phylogenetics
+- genbank
+- dna
+- sequences
+- data-access
+- taxonomy
+---
+
+## What is phylotaR?
+
+In any phylogenetic analysis it is important to identify sequences that share the same orthology. This is often performed by searching an online sequence repository, such as GenBank, using a gene/region name and the taxa of interest, e.g. 'Aotus' and 'cytochrome oxidase'. Relying solely on sequence labels, however, can miss sequences that have either not been labelled, have unanticipated names or have been mislabelled.
+
+The phylotaR R package, like its earlier inspiration [PhyLoTa](http://phylota.net/), identifies sequences of shared orthology not by naming matching but instead through the use of an alignment search tool, [BLAST](https://en.wikipedia.org/wiki/BLAST). As a result, a user of phylotaR will able to download more relevant sequence data for their phylogenetic analysis than they otherwise would.
+
+The entire phylotaR pipeline is automated and all a user needs to supply is a taxonomic group of interest for which the user would like to identify ortholgous sequences. The pipeline is broken down into four stages: retrieve taxonomic information, download sequences, identify clusters, identify clusters among the clusters.
+
```

Comment:
In figure `stages.png` when you refer to "Taxise" I assume you are referring to a general stage of getting taxonomic info, not the `taxize` package

Reply:
That is correct.

In hindsight, I should never have called the stage taxise, it just creates confusion.

Reply:
I'll add an explainer under the stages.png